source/elements/oneTBB/source/algorithms/functions/parallel_pipeline_func/filter_cls.rst

```diff
@@ -48,9 +48,11 @@ The ``filter`` class should only be used in conjunction with ``parallel_pipeline
 
 Requirements:
 
-* If `InputType` is ``void``, a ``Body`` type must meet the :doc:`StartFilterBody requirements <../../../named_requirements/algorithms/filter_body>`.
-* If `OutputType` is ``void``, a ``Body`` type must meet the :doc:`OutputFilterBody requirements <../../../named_requirements/algorithms/filter_body>`.
+* If `InputType` is ``void``, a ``Body`` type must meet the :doc:`FirstFilterBody requirements <../../../named_requirements/algorithms/filter_body>`.
+* If `OutputType` is ``void``, a ``Body`` type must meet the :doc:`LastFilterBody requirements <../../../named_requirements/algorithms/filter_body>`.
+  Alternatively, since C++17 a pointer to member function in ``InputType`` returning ``OutputType`` might be used as a ``Body``.
```

Comment:
```suggestion
  Alternatively, since C++17 a pointer to a member function in ``InputType`` returning ``void`` might be used as a ``Body``.
```
Seems it should be like that, otherwise there is a duplication with a similar line below.